### ISSUE TITLE
Load disc image from local 

### ISSUE THREAD

**Issue by 8bitkick** [pdjstone/archimedes-live#19]
Is there a way to load an .adf image from my machine?

PS Thanks for the thanks. Love this emulator and the fact the default is the A3000, the machine which got me started on ARM!

**Comment by pdjstone** [comment]
Yes! Although there's absolutely no UI indication it's possible (see issue #7) you can drag-drop a local .adf file anywhere on the page and it should get inserted into drive :0. You can also drop ArcFS / SparkFS files and they'll be unpacked onto HostFS.

**Comment by pdjstone** [comment]
I've added an upload button to the software browser dialogue, which should make it a bit more obvious that this is possible.
